src/core/annotation.js

```diff
@@ -1282,6 +1285,15 @@ class WidgetAnnotation extends Annotation {
     const totalHeight = this.data.rect[3] - this.data.rect[1];
     const totalWidth = this.data.rect[2] - this.data.rect[0];
 
+    if (!this.data.defaultAppearance) {
+      // The DA is required and must be a string.
+      // There is no reason to have a font named Helvetica in
```

Comment:
Why not? The PDF file might just have a font named `Helvetica` in the dictionary. I think this whole sentence should be rephrased to: `If there is no font named Helvetica in the resource dictionary, the evaluator will fall back to a default font. Doing so prevents exceptions and allows saving/printing the file as expected.`